controller/api/models.py

```diff
@@ -251,9 +249,13 @@ def restart(self, **kwargs):
 
     def _clean_app_logs(self):
         """Delete application logs stored by the logger component"""
-        path = os.path.join(settings.DEIS_LOG_DIR, self.id + '.log')
-        if os.path.exists(path):
-            os.remove(path)
+        try:
+            url = 'http://{}:{}/{}/'.format(settings.LOGGER_HOST, settings.LOGGER_PORT, self.id)
+            requests.delete(url)
+        except Exception:
```

Comment:
http://c2.com/cgi/wiki?CatchWhatYouCanHandle

In this case, if something unrelated like a BufferError or an IOError occurs, we want to make sure that those errors are not swallowed by this exception and let Django handle the exception.